docs-compose/overwatch_docs/content/DataEngineer/Modules.md

```diff
@@ -250,4 +251,11 @@ Additional clarification of the Spark Hierachy can be found on this [Spark+AI Su
   though a task can be retried because a unique Task is also identified by the AttemptID; thus, when a task fails on
   due to node failure, the task + attempt is marked as failed and a new attempt is executed on another node. The
   ExecutorID is the node on which the work was executed.
-  ![TaskViz](/images/GettingStarted/Modules/Task_Detail.png)
\ No newline at end of file
+  ![TaskViz](/images/GettingStarted/Modules/Task_Detail.png)
+
+### VerboseAuditLogging
+*Requires:* Audit|Notebook|ClusterEvents
+
+*Gold Entities:* NotebookCommands
+
+To enable this module user need to enable verbose logging in their workspace. This module that enables the materialization of notebooks commands with cost dimension related to the notebook commands.
```

Comment:
Change to: "To enable this module the user needs to enable verbose audit logging in their workspace. This module enables the materialization of notebooks commands with cost dimension related to the notebook commands."

Reply:
Done